web-crawler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -80,11 +80,13 @@ public void compileNewsArticle(String candidateName, String candidateId) {
         continue;
       }
       NewsArticle newsArticle = potentialNewsArticle.get();
+      RelevancyChecker relevancyChecker;
       try {
-        if (!RelevancyChecker.isRelevant(newsArticle, candidateName)) {
-          continue;
-        }
-      } catch (Exception e) {
+        relevancyChecker = new RelevancyChecker();
+      } catch (IOException e) {
+        continue;
+      }
```

Comment:
Can you instantiate this outside of the for loop, once for this function? (or better yet, make it an instance variable so that WebCrawler doesn't keep creating this object)